include/deal.II/lac/block_vector.h

```diff
@@ -517,6 +518,16 @@ namespace internal
   } /* namespace LinearOperator */
 } /* namespace internal */
 
+
+/** mark dealii::BlockVector< Number > as serial vector
```

Comment:
Ok, well, let me just double check how single line comments are done in other parts of the documentation. Give me a moment...